Cargo.toml

```diff
@@ -91,6 +91,7 @@ check-cfg = [
   'cfg(getrandom_test_linux_fallback)',
   'cfg(getrandom_test_linux_without_fallback)',
   'cfg(getrandom_test_netbsd_fallback)',
+  'cfg(target_os, values("cygwin"))'
```

Comment:
Please remove this line

Reply:
Will the compiler warn or error if this line is removed?

Reply:
It will not, note that we don't have any other entries for `target_os` despite it being used in this crate. 

For why this is the case, see https://doc.rust-lang.org/rustc/check-cfg.html#well-known-names-and-values. `target_os` is a "well-known name"

Reply:
`target_os` is a well known name, but "cygwin" is not a well known value before rust 1.86. I'll try to remove thia line.

Reply:
I was absolutely wrong here, you were correct this line is necessary. The `unexpected_cfgs` is very annoying and confusing.